manifests/agent/yum_repos.pp

```diff
@@ -0,0 +1,11 @@
+class classroom::agent::yum_repos {
+  $classroom::yum_repos.each |$repo, $value| {
```

Comment:
classroom::yum_repos - where is it defined and was this meant to be classroom_repos as in manifests/master/yum_server.pp?